src/sentry/tagstore/snuba/backend.py

```diff
@@ -599,7 +600,7 @@ def get_tag_value_paginator_for_projects(
             snuba_key = "tags[%s]" % (key,)
 
         if query:
-            conditions.append([snuba_key, "LIKE", u"%{}%".format(query)])
+            conditions = get_filter(query).conditions
```

Comment:
This looks risky to me. The query parser won't generate LIKE conditions unless the value has `*` on each side.